controller/main.php

```diff
@@ -91,6 +91,13 @@ public function fetchinfo($type, $fingerprint)
 				$yaml .=  "\tProfileID: " . $data['user_id'] . "\n";
 				$yaml .=  "\tProfileName: " . $data['username'] . "\n";
 				$yaml .=  "\tProfileRank: Registered User\n";
+				$yaml .=  "\tAvatar:\n";
+				if ($avatar_data = $this->core->get_avatar_data($data))
+				{
+					$yaml .=  "\t\tSrc: {$avatar_data['src']}\n";
```

Comment:
Sorry for not noticing and commenting on this before, but can we please stick with string concatenation for consistency with the lines above?